VotingApplication/VotingApplication.Web/Scripts/Controllers/HomePageController.js

```diff
@@ -1,10 +1,65 @@
 ﻿(function () {
     var VotingApp = angular.module('VotingApp');
 
-    VotingApp.controller('HomePageController', function ($scope) {
+    VotingApp.controller('HomePageController', ['$scope', 'PollService', function ($scope, PollService) {
         $scope.models = {
             pageTitle: 'Pollster'
         };
-    });
+
+        $scope.pollExpiry = undefined;
+
+        var timeStringForTimeout = function (expiryDate, timeUnit, timeUnitName) {
```

Comment:
Both this and the calculateExpiry functions could probably be moved out of this controller. I anticipate that the poll management page might want to also show this information at a later date.